cdc/owner.go

```diff
@@ -719,6 +719,8 @@ func (c *changeFeed) calcResolvedTs() error {
 
 	if len(c.taskPositions) == 0 {
 		minCheckpointTs = c.status.CheckpointTs
+	} else if len(c.taskPositions) < len(c.taskStatus) {
```

Comment:
`len(c.taskPositions) < len(c.taskStatus)` can detect the scenario that a new processor which hasn't written `task position` to etcd.
But what happens when a processor already exists, and it is assigned a new table and the checkpoint ts of the newly added table is not considered